packages/ui/components/head-seo/HeadSeo.tsx

```diff
@@ -129,6 +132,7 @@ export const HeadSeo = (props: HeadSeoProps): JSX.Element => {
     openGraph: {
       ...nextSeoProps.openGraph,
       ...seoObject.openGraph,
+      title: APP_NAME,
```

Comment:
I'd suggest removing the translation from the title as that cannot be server side rendered with the current `next-seo` setup. This story is different once you are on app router where you can make use of [`generateMetadata`](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#generatemetadata-function) and have access to the server for setting `<head/>` content.